examples-new/webhooks/templates/index.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% block body %}
+<p>This example doesn't have anything to see in the browser. Set up your
```

Comment:
Can you link to Ngroks web interface please? It's nice to replay POSTs from Nylas this way without needing to actually re-create a triggered notification.